scripts/deb-builder-helpers.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/bash
+
+# Helper script to include when building deb archives.
+
+echo "--- Setting up the envrionment to build debian packages..."
+
+set -euo pipefail
+
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+cd "${SCRIPTPATH}/../_build"
+
+GITHASH=$(git rev-parse --short=7 HEAD)
+GITHASH_CONFIG=$(git rev-parse --short=8 --verify HEAD)
+
+set +u
+BUILD_NUM=${BUILDKITE_BUILD_NUM}
```

Comment:
this script is outside buildkite folder so IMHO it should not know about any buildkite existence

Reply:
The existing `scripts/rebuild-deb.sh` also has this reference even after your recent refactor. I'd prefer to make this change in another PR, as it's unrelated to this work and has been this way int he code for some time now.